src/core/Akka.Remote.TestKit/Internals/Reference.conf

```diff
@@ -34,7 +34,7 @@ akka {
       # (I&O) Used to configure the number of I/O worker threads on server sockets
       server-socket-worker-pool {
         # Min number of threads to cap factor-based number to
-        pool-size-min = 1
+        pool-size-min = 2
```

Comment:
This hocon is for Akka.Remote which doesn't use Helios 2.0 yet isn't? Increasing the pool size might not cause reordering issue?